content/en/monitors/notify/variables.md

```diff
@@ -322,6 +322,7 @@ To include **any** attribute or tag from a log, a trace span, a RUM event, a CI
 | Trace Analytics | `{{span.attributes.key}}` or `{{span.tags.key}}` |
 | Error Tracking  | Traces: `{{span.attributes.[error.message]}}`<br>RUM Events: `{{rum.attributes.[error.message]}}`<br>Logs: `{{log.attributes.[error.message]}}`             |
 | RUM             | `{{rum.attributes.key}}` or `{{rum.tags.key}}`   |
+| Audit Trail     | `{{audit.attributes.key}}` or `{{audit.message}}`    |
```

Comment:
Audit Trail seems to be missing from the "Available for" section above the table 
![2023-11-07-at-10 52 45@2x](https://github.com/DataDog/documentation/assets/6892255/8690860e-0169-4e64-a8d2-82988518ec39)